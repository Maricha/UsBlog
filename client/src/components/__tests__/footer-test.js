import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';
import Footer from '../footer';

describe('Footer Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Footer />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  })
});