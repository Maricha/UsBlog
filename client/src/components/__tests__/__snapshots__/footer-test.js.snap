// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders correctly 1`] = `
<footer
  className="Component-footer-1"
>
  <div
    className="Component-content-3"
  >
    <h6
      className="MuiTypography-root-6 MuiTypography-h6-23 MuiTypography-gutterBottom-33 MuiTypography-alignCenter-29"
    >
      Marcin Starmach
    </h6>
    <p
      className="MuiTypography-root-6 MuiTypography-subtitle1-24 MuiTypography-colorTextSecondary-39 MuiTypography-alignCenter-29"
    >
      Blog
    </p>
  </div>
  <div
    className="Component-social-2"
  >
    <div
      className="Component-socialIcon-4"
    />
    <div
      className="Component-socialIcon-4"
    />
  </div>
</footer>
`;
